pkg/server/mysql/mysql.go

```diff
@@ -40,13 +40,13 @@ func (p *MySQLPlugin) DatabasePlugin() plugin.DatabasePlugin {
 	return p
 }
 
-func (p *MySQLPlugin) Dsn(pw string, s *atlas.DatabaseServer) string {
+func (p *MySQLPlugin) Dsn(userName string, password string, db *atlas.Database, s *atlas.DatabaseServer) string {
 	return fmt.Sprintf("%s:%s@tcp(%s.%s:%d)/mysql?charset=utf8&parseTime=True",
-		s.Spec.SuperUser,
-		pw,
+		userName,
+		password,
 		s.Name,
 		s.Namespace,
-		s.Spec.Port)
+		s.Spec.ServicePort)
```

Comment:
this should be `Port` or `ServicePort` ?

Reply:
I just sticked with ServicPort so removed Port